packages/widget_toolkit_pin/lib/src/lib_pin_code_with_biometrics/blocs/pin_code_bloc.dart

```diff
@@ -54,36 +98,61 @@ class PinCodeBloc extends $PinCodeBloc {
     var biometricsEnabled =
         await biometricAuthenticationService.areBiometricsEnabled();
 
-    var pinCode = await pinCodeService.getPinCode();
-    return isDeviceSupported &&
-        canCheckBiometrics &&
-        biometricsEnabled &&
-        pinCode != null;
+    return isDeviceSupported && canCheckBiometrics && biometricsEnabled;
```

Comment:
I believe the biometrics should be available only if the pin is already stored, otherwise what you would send to the server ?

Reply:
I have returned this check